### PR TITLE
feat: super basic gql typeDefs/resolver and schema-driven ts gql types

### PR DESCRIPTION
**Pull request by patrickocoffeyo**

### CODE REVIEW

File: src/lib/graphql/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import type { Resolvers } from '@graphql-poc/graphql';
+import { join } from 'path';
+import { importSchema } from 'graphql-import';
+import { makeExecutableSchema } from 'graphql-tools';
+import * as Query from './resolvers/Query';
+import { GraphQLSchema } from 'graphql';
+
+// @TODO: This will need to be fleshed out when we have more requirements.
```
Comment (chasingmaxwell) [inline src/lib/graphql/index.ts:8]: Even though we're not using it currently, This should still probably set to what this value is currently. Even an empty object would be better than `any`.

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const host = 'localhost';
+  app.listen({ port }, () => {
+    /* tslint:disable no-console */
+    console.info(
+      `😎 Server available at ${host}:${port}${server.graphqlPath}, and playground is available at ${host}:${port}${server.graphqlPath}/playground`
+    );
+  });
+}
+
+/* tslint:disable no-floating-promises */
```
Comment (chasingmaxwell) [inline src/server.ts:22]: are we using tslint? I think we can just use eslint with ts now.

File: src/lib/graphql/resolvers/Query/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const wizard = (): string => 'nice!';
```
Comment (chasingmaxwell) [inline src/lib/graphql/resolvers/Query/index.ts:1]: If you return something other than a string, does that cause type errors? Just making sure all the types are hooked up appropriately.

Comment (patrickocoffeyo) [reply src/lib/graphql/resolvers/Query/index.ts:1]: Yes, changing the return type results in type errors in the code that composes the gql schema.

**Review by chasingmaxwell — APPROVED** [review]

**Pull request merged**